src/utilities/host-utils.ts

```diff
@@ -0,0 +1,12 @@
+export const isOfflineHost = (host: string) => (host === "localhost:11002") || (host === "activity-player-offline.concord.org");
```

Comment:
Does anyone use `127.0.0.1` when doing development?  
The localhost check is just for development.
Because this will trigger a service worker setup on each hostname, it won't work to switch between localhost and `127.0.0.1`, all of the resources would be cached on each domain individually.

I'd vote to just stick with localhost.
But I'll leave it up to @dougmartin